operator/Makefile

```diff
@@ -21,7 +21,7 @@ LOKI_OPERATOR_NS ?= kubernetes-operators
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= v0.3.0
+VERSION ?= v0.4.0
```

Comment:
I can only see `$VERSION` used together with `subst` in this file... do we use it anywhere else that I'm missing? If not, we could just remove the `v` and get rid of the `subst`.